mass_mailing_list_dynamic/__manifest__.py

```diff
@@ -15,6 +15,7 @@
     "depends": ["mass_mailing_partner"],
     "data": [
         # This should go first
+        "security/ir.model.access.csv",
```

Comment:
You are deforming the comment meaning (which should be by the way `This should go before "mailing_list_view.xml"`)

Reply:
You have put it in other bad place and not changing the comment